common/common.go

```diff
@@ -233,6 +235,16 @@ const (
 	EnvCMPWorkDir = "ARGOCD_CMP_WORKDIR"
 	// EnvGPGDataPath overrides the location where GPG keyring for signature verification is stored
 	EnvGPGDataPath = "ARGOCD_GPG_DATA_PATH"
+	// EnvServerName is the pod selector labels of the Argo CD server component.
+	EnvServerName = "ARGOCD_SERVER_NAME"
+	// EnvRepoServerName is the pod selector labels of the Argo CD repo server component.
+	EnvRepoServerName = "ARGOCD_REPO_SERVER_NAME"
+	// EnvAppControllerName is the pod selector labels of the Argo CD application controller component.
+	EnvAppControllerName = "ARGOCD_APPLICATION_CONTROLLER_NAME"
+	// EnvRedisName is the pod selector labels of the Argo CD redis component.
+	EnvRedisName = "ARGOCD_REDIS_NAME"
+	// EnvRedisHaHaproxyName is the pod selector labels of the Argo CD Redis HA proxy component.
+	EnvRedisHaHaproxyName = "ARGOCD_REDIS_HAPROXY_NAME"
```

Comment:
Should we represent in the comments that the env variable holds the custom name of the component and not the pod selector label?